blog/2024-11-2-app-platform-v12.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: 2024/11/app-platform-v12
+title: App Platform v12
+authors: core
+tags: [app platform, developer tools, webapp, announcement]
+---
+
+# Platform v12 Migration Guide: Vite & React 18
+
+Good news! Vite and React 18 in the app platform are ready to use!
+
+We're very excited for these updates — both will significantly modernize the App Platform, and Vite will be a big upgrade from Create React App (CRA), which we used under the hood to build and serve apps previously. It will greatly improve the developer experience, and with greater control over the configuration, it will open up some powerful new possibilities for the platform.
```

Comment:
I'm suggesting this, as my previous suggestion already explained CRA. 

```suggestion
We're very excited for these updates — both will significantly modernize the App Platform, and Vite will be a big upgrade from CRA, which we used under the hood to build and serve apps previously. It will greatly improve the developer experience, and with greater control over the configuration, it will open up some powerful new possibilities for the platform.
```